sources/web/datalab/polymer/modules/file-manager/file-manager.ts

```diff
@@ -71,21 +71,21 @@ interface FileManager {
    *               useful for downloading notebooks, which are by default read
    *               as JSON, which doesn't preserve formatting.
    */
-  getFile(path: string, asText?: boolean): Promise<DatalabFile>;
+  get(path: string, asText?: boolean): Promise<DatalabFile>;
 
   /**
    * Uploads the given file object to the backend. The file's name, path, format,
    * and content are required fields.
    * @param model object containing file information to send to backend
```

Comment:
doc name (model) does not match function param name (file).
Should this return Promise<boolean> like delete, or is there additional information it needs to return?
If so, this could be done in a separate PR.